supabase/migrations/44_draft_associated_collections.sql

```diff
@@ -0,0 +1,67 @@
+alter policy "Users must be authorized to referenced specifications" on live_spec_flows
+  rename to "Users must be authorized to one referenced specification";
+
+alter policy "Users must be authorized to one referenced specification" on live_spec_flows
+  using (
+    source_id in (select id from live_specs) or
+    target_id in (select id from live_specs)
+  );
+
+create or replace function internal.get_collections_eligible_for_deletion(capture_id flowid)
+returns table(id flowid, catalog_name catalog_name, last_pub_id flowid) as $$
+begin
+
+return query with target_collections as (
+  select target_id from live_spec_flows
+    where source_id = capture_id
+),
+collections_read as (
+  select target_collections.target_id from target_collections
+    join live_spec_flows lsf on target_collections.target_id = lsf.source_id
+),
+collections_written as (
+  select target_collections.target_id from target_collections
+    join live_spec_flows lsf on target_collections.target_id = lsf.target_id and lsf.source_id <> capture_id
+),
+ineligible_collections as (
+  select target_id from collections_read
+    union select target_id from collections_written
+),
+eligible_collections as (
+  select target_id from target_collections
+    except select target_id from ineligible_collections
+)
+select
+  ls.id,
+  ls.catalog_name,
+  ls.last_pub_id
+from eligible_collections
+  join live_specs ls on eligible_collections.target_id = ls.id;
+
+end;
+$$ language plpgsql security definer;
```

Comment:
Now that the RLS policy on `live_spec_flows` is updated, shouldn't this be `security invoker`?